.gitignore

```diff
@@ -29,6 +29,10 @@ build/
 .gradle
 local.properties
 *.iml
+**/fastlane/report.xml
+**/fastlane/Preview.html
+**/fastlane/screenshots
+**/fastlane/test_output
```

Comment:
It's duplicated.
Check this out: https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/master/.gitignore#L54.
Maybe we need to add `**/` at the start of the others, can you try?